packages/shared/src/styles/themes.scss

```
@@ -89,6 +90,7 @@
         --text-red: #{$color-red};
         --text-blue: #{$color-blue-3};
         --text-info-blue: #{$color-blue};
+        --text-info-blue-bg: #{$color-blue-5};
```

```suggestion
        --text-info-blue-background: #{$color-blue-5};
```
Let's follow a standard naming practice like the rest